File: LNPG_01_TIPOS_E_VARIAVEIS/Java/src/par_ou_impar_2.java
import java.util.Scanner;

public class par_ou_impar_2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Insira um numero inteiro e verifique se ele e par ou impar.\n");
        int number_user = scanner.nextInt();

        if (number_user % 2 == 0) {
            System.out.printf("O numero %d e par", number_user);
        } 
        else {
            System.out.printf("O numero %d e impar", number_user);
        }

        scanner.close();
    }
}

File: LNPG_01_TIPOS_E_VARIAVEIS/C/par_ou_impar_2.c
#include <stdio.h>

int main() {
    int number_user;
    printf("Insira um numero inteiro e verifique se ele e par ou impar.\n");
    scanf("%d", &number_user);
    
    if (number_user % 2 == 0) {
        printf("O numero %d e par.", number_user);
    } 
    else {
        printf("O numero %d e impar.", number_user);
    }
    
    return 0;
}